# GetState.srv
---
string current_state

